src/fde.h

```diff
@@ -97,6 +114,18 @@ class fde
     /// record a transaction on this FD
     void noteUse() { ++pconn.uses; }
 
+    /// record bytes being read and return the accumulated total for this FD.
```

Comment:
This description got stale -- the method no longer returns anything. We should also polish it to emphasize that Squid read the bytes in question (rather than being in the process of reading them) -- the I/O being recorded here is over. For example:

```suggestion
    /// Record bytes received from a single I/O call.
    /// Whenever possible, configure I/O methods and use read() instead.
```